Description: Use pkg-config to detect FreeType 2
 Use pkg-config to detect FreeType 2, as freetype-config is not installed 
 in FreeType 2.9.1 or later.
Author: Hugh McMaster <hugh.mcmaster@outlook.com>
Bug-Debian: https://bugs.debian.org/887606
Last-Update: 2018-11-10

--- dvipng-1.15/configure.ac
+++ dvipng-1.15/configure.ac
@@ -78,12 +78,21 @@ AC_CHECK_HEADERS([gd.h png.h kpathsea/kp
 AC_CHECK_HEADERS([libgen.h])
 PSFONTS_O=""
 AC_SUBST(PSFONTS_O)
-AC_CHECK_FT2(,[CFLAGS="$FT2_CFLAGS $CFLAGS"
-	LIBS="$FT2_LIBS $LIBS"
-	PSFONTS_O="sfd.o ft.o enc.o fontmap.o tfm.o"
-	AC_DEFINE(HAVE_FT2, 1, [Define to 1 if you have freetype2])
-	ac_have_freetype2="yes"],
-	[ac_have_freetype2="no"])
+
+PKG_PROG_PKG_CONFIG([0.29])
+
+if test -z "$PKG_CONFIG"; then
+    ac_have_freetype2 = "no"
+else
+    PKG_CHECK_MODULES([FT2], [freetype2], [ac_have_freetype2="yes"], [ac_have_freetype2="no"])
+    if test "$ac_have_freetype2" = "yes"; then
+        CFLAGS="$FT2_CFLAGS $CFLAGS"
+        LIBS="$FT2_LIBS $LIBS"
+        PSFONTS_O="sfd.o ft.o enc.o fontmap.o tfm.o"
+        AC_DEFINE([HAVE_FT2], [1], [Define to 1 if you have freetype2])
+    fi
+fi
+
 AC_HEADER_SYS_WAIT
 AC_HEADER_TIME
 AC_HEADER_STDBOOL
